### ISSUE TITLE
Compile errors with Visual C++ 2008 Express Edition

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#561]
**Reported by hmonroe on 20 Mar 38237249 21:46 UTC**
I get the following compile errors with Visual C++ 2008 Express Edition when compiling the solutions samples.sln. 
```
cannot find GL/glaux.h
cannot find glaux.lib
error C2664: 'BOOL (HANDLE,PSYM_ENUMMODULES_CALLBACK64,PVOID)' : cannot convert parameter 2 from 'overloaded-function' to 'PSYM_ENUMMODULES_CALLBACK64'
```

**Comment by romw** [comment]
**Commented by Nicolas on 31 Aug 38237290 19:06 UTC**
argh

**Comment by romw** [comment]
**Commented by S@NL - FilmFreak on 12 Mar 38239195 05:20 UTC**
Unpack the contents of GL.rar in a folder called GL in the boinc folder. For the error see http://boinc.berkeley.edu/trac/ticket/115

**Comment by romw** [comment]
**Commented by romw on 13 Aug 39830766 16:26 UTC**
Fixed in trunk and 6.10 branches
